plugins/sigma.renderers.edgeLabels/sigma.canvas.edges.labels.curve.js

```diff
@@ -44,13 +45,13 @@
         t = 0.5;  //length of the curve
 
     if (source.id === target.id) {
-      cp = sigma.utils.getSelfLoopControlPoints(sX, sY, sSize);
+      cp = sigma.utils.getSelfLoopControlPoints(sX, sY, sSize, count);
```

Comment:
I may be mistaken but reading the code for both functions `getSelfLoopControlPoints` and `getQuadraticControlPoint` don't accept this last argument. Is this function overwritten somewhere I can't find?